### PR TITLE
Some improvements on the 2.6 migration guide

### PR DESCRIPTION
**Pull request by octonato**
* fork-join pool paragraph move out artery section
 * unnesting artery section
 * being explicit how to disable artery

### CODE REVIEW

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 1]
```diff
@@ -126,20 +126,19 @@ blocking and protect a bit against starving the internal actors. Since the inter
 the default dispatcher has been adjusted down to `1.0` which means the number of threads will be one per core, but at least
 `8` and at most `64`. This can be tuned using the individual settings in `akka.actor.default-dispatcher.fork-join-executor`.
 
+## Akka now uses Fork Join Pool from JDK
+
+Previously, Akka contained a shaded copy of the ForkJoinPool. In benchmarks, we could not find significant benefits of keeping our own copy, so from Akka 2.6 on, the default FJP from the JDK will be used. The Akka FJP copy was removed.
```
Comment (helena) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]:
```suggestion
Previously, Akka contained a shaded copy of the ForkJoinPool. In benchmarks, significant benefits of keeping our own copy were not found. From Akka 2.6 on, the default FJP is from the JDK and the Akka FJP copy is removed.
```

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 2]
```diff
@@ -194,9 +193,9 @@ For TCP:
 * `akka.remote.artery.advanced.connection-timeout` to `akka.remote.artery.advanced.tcp.connection-timeout`
 
 
-#### Remaining with Classic remoting (not recommended)
+### Remaining with Classic remoting (not recommended)
 
-Classic remoting is deprecated but can be used in `2.6.` Any configuration under `akka.remote` that is
+Classic remoting is deprecated but can be used in `2.6.` First you need to explicitly disable Artery by setting property `akka.remote.artery.enabled` to `false`. Further, any configuration under `akka.remote` that is
```
Comment (helena) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]:
```suggestion
Classic remoting is deprecated but can be used in `2.6.` Explicitly disable Artery by setting property `akka.remote.artery.enabled` to `false`. Further, any configuration under `akka.remote` that is
```

**Review by helena — COMMENTED** [review]
Nice addition

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 3]
```diff
@@ -169,7 +168,7 @@ If using SSL then `tcp-tls` needs to be enabled and setup. See @ref[Artery docs
 for how to do this.
 
 
-#### Migration from 2.5.x Artery to 2.6.x Artery
+### Migration from 2.5.x Artery to 2.6.x Artery
```
Comment (patriknw) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]: note that there were some structural changes in https://github.com/akka/akka/pull/27025
is this still correct?

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md (hunk 2 above)
Comment (patriknw) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]: good catch that the most essential part was missing

**Comment by renatocaval** [comment]
I will send another PR. Some of the changes here were done in #27025.

**Pull request closed without merging**